hr_recruitment_candidate_multi_applicant/tests/test_multi_applicant.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.tests import common
+
+
+class TestApplicant(common.TransactionCase):
+
+        def setUp(self):
+            super(TestApplicant, self).setUp()
+            self.hr_applicant = self.env['hr.applicant']
+            self.toto = self.hr_applicant.create({
+                'name': 'Toto',
+                'email_from': 'toto@camptocamp.com',
+            })
+            self.toto2 = self.hr_applicant.create({
+                'name': 'Toto2',
+                'email_from': 'toto@camptocamp.com',
+            })
+            self.jdoe = self.hr_applicant.create({
+                'name': 'John Doe',
+                'email_from': 'jdoe@camptocamp.com',
+            })
+
+        def test_compute_same_email(self):
+            self.assertEqual(1, self.toto.same_candidate_application_count)
+            self.assertEqual(1, self.toto2.same_candidate_application_count)
+            self.assertEqual(0, self.jdoe.same_candidate_application_count)
+
+        def test_same_phone(self):
```

Comment:
Same with `test_compute_same_email`?

Reply:
@hieulucky111 I don't undersand your point. The email is tested in => https://github.com/OCA/hr/pull/373/files/fa410c9efb6192e0ce51a251679d122e6d61e033#diff-59fe9f77b440f08b6354e3e4b21b3720R26